gbus/messages.go

```diff
@@ -118,3 +118,27 @@ type SagaTimeoutMessage struct {
 func (SagaTimeoutMessage) SchemaName() string {
 	return "grabbit.timeout"
 }
+
+// In order to route "resurrected" events to the correct handler it is necessary to use the original exchange and routing-key which
+// were added by grabbit previously as headers, as opposed to the native values on the delivery
+// Issue reference:	https://github.com/wework/grabbit/issues/191
+func exchangeAndRoutingFromDelivery(delivery amqp.Delivery) (exchange string, routingKey string, err error) {
+	if isResurrectedMessage(delivery) {
```

Comment:
this is a weird name. you're going to resurrect something but your'e checking if it's resurrected... I think you should change the name of the method and header

Reply:
@yuvmendel I think the comments you added outside of the method  should be placed inside just before the 
if isResurrectedMessage.
In addition, please see @danielwitz question as an indicator that this documentation is not clear enough or doesn't provide enough details as to what is going on in this method.

Reply:
yea you are right, it's more of an implementation explanation, ill move it.
I think maybe ill also rename this method to "getRoutingParamsFromDelivery"

I wonder how to further make this more obvious, I think perhaps this should go back to the worker.go file - this is a private method which is used in a very specific part of the flow and was separated for readability, and it is not really relevant to messages in general.
I think perhaps having it here caused the confusion regarding the context of the method and at which part is it called.

@rhinof WDYT think about this renaming and moving the methods back to worker.go?

Reply:
@yuvmendel I think you are spot-on, moving this back to worker.go to the context it is being used at is a good idea. the name you proposed is fine as well.
Think once we resolve this issue we can go ahead and merge this PR